aodncore/pipeline/statequery.py

```diff
@@ -66,4 +53,22 @@ def query_wfs_url_exists(self, layer, name):  # pragma: no cover
         :param name: 'file_url' inserted into OGC filter, and supplied to GetFeature filter parameter
         :return: list of files for the layer
         """
-        return self._wfs_broker.query_url_exists_for_layer(layer, name)
+        return self._wfs_broker.query_url_exists(layer, name)
+
+    # Storage methods
+    def download(self, remotepipelinefilecollection, local_path):
+        """Helper method to download a RemotePipelineFileCollection or RemotePipelineFile
+
+        :param remotepipelinefilecollection: RemotePipelineFileCollection to download
+        :param local_path: local path where files will be downloaded. Defaults to the handler's :attr:`temp_dir` value.
```

Comment:
I don't see where the default to the handler's `temp_dir` is implemented. Should that be in here?

Reply:
Copy/paste from when the method was in HandlerBase. Fixed by removing it.